Source/Core/Core/State.cpp

```diff
@@ -98,7 +98,7 @@ static size_t s_state_writes_in_queue;
 static std::condition_variable s_state_write_queue_is_empty;
 
 // Don't forget to increase this after doing changes on the savestate system
-constexpr u32 STATE_VERSION = 167;  // Last changed in PR 12494
+constexpr u32 STATE_VERSION = 168;  // Last changed in PR 12702
```

Comment:
I don't think you need to change this. The DoState in GameFile.cpp is linked to CACHE_REVISION, not STATE_VERSION.